<ui version="4.0" >
 <class>NetworkConsole</class>
 <widget class="QDialog" name="NetworkConsole" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>366</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Network Details</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="2" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QTabWidget" name="tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="neuronsTab" >
      <attribute name="title" >
       <string>Neurons</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="activationFunctionTab" >
      <attribute name="title" >
       <string>Activation function</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="resilientPropagationTab" >
      <attribute name="title" >
       <string>Resilient propagation</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
